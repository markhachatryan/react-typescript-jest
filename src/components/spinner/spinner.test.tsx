import * as React from "react";
import Spinner from "./spinner";
import * as renderer from "react-test-renderer";

test("test for react test", () => {
    const component = renderer.create(<Spinner />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
