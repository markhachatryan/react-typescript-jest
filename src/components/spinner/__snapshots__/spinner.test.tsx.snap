// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for react test 1`] = `
<div
  className="lds-css ng-scope"
>
  <div
    className="lds-blocks"
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <div
      style={
        Object {
          "animationDelay": "0s",
          "left": "38px",
          "top": "38px",
        }
      }
    />
    <div
      style={
        Object {
          "animationDelay": "0.125s",
          "left": "80px",
          "top": "38px",
        }
      }
    />
    <div
      style={
        Object {
          "animationDelay": "0.25s",
          "left": "122px",
          "top": "38px",
        }
      }
    />
    <div
      style={
        Object {
          "animationDelay": "0.875s",
          "left": "38px",
          "top": "80px",
        }
      }
    />
    <div
      style={
        Object {
          "animationDelay": "0.375s",
          "left": "122px",
          "top": "80px",
        }
      }
    />
    <div
      style={
        Object {
          "animationDelay": "0.75s",
          "left": "38px",
          "top": "122px",
        }
      }
    />
    <div
      style={
        Object {
          "animationDelay": "0.625s",
          "left": "80px",
          "top": "122px",
        }
      }
    />
    <div
      style={
        Object {
          "animationDelay": "0.5s",
          "left": "122px",
          "top": "122px",
        }
      }
    />
  </div>
</div>
`;
